codemodes/i18next/codemod.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-param-reassign */
+function replaceTranslateImport(file, j) {
+	let foundAndReplaced = false;
+	const result = j(file.source)
+		.find(j.ImportDeclaration, {
+			source: {
+				value: 'react-i18next',
+			},
+		})
+		.forEach(nodePath => {
+			nodePath.value.specifiers = nodePath.value.specifiers.map(specifier => {
+				if (specifier.type === 'ImportSpecifier' && specifier.imported.name === 'translate') {
+					foundAndReplaced = true;
+					return j.importSpecifier(j.identifier('withTranslation'));
+				}
+				return specifier;
+			});
+		})
+		.toSource();
+
+	return foundAndReplaced ? result : null;
+}
+
+function replaceTranslateCalls(file, j) {
+	return j(file)
+		.find(j.CallExpression, { callee: { name: 'translate' } })
+		.forEach(nodePath => {
+			nodePath.value.callee = j.identifier('withTranslation');
+		})
+		.toSource();
```

Comment:
In other code mode we call toSource with the following options:
```
toSource({ useTabs: true, quote: 'single', trailingComma: true })
```

why this is not the case here ?

Reply:
Oh yes thanks !